backend/settings/base.py

```diff
@@ -49,6 +50,8 @@ class Base(Configuration):
     INSTALLED_APPS = DJANGO_APPS + THIRD_PARTY_APPS + LOCAL_APPS
 
     MIDDLEWARE = [
+        "corsheaders.middleware.CorsMiddleware",
```

Comment:
I know this must go first but I believe `django.middleware.common.CommonMiddleware` can stay where it was